examples/widgets/check_buttons.py

```diff
@@ -8,23 +8,25 @@
 s2 = np.sin(6*np.pi*t)
 
 fig, ax = plt.subplots()
-l0, = ax.plot(t, s0, visible=False, lw=2)
-l1, = ax.plot(t, s1, lw=2)
-l2, = ax.plot(t, s2, lw=2)
+ax.plot(t, s0, visible=False, lw=2, color='k', label='2 Hz')
+ax.plot(t, s1, lw=2, color='r', label='4 Hz')
+ax.plot(t, s2, lw=2, color='g', label='6 Hz')
 plt.subplots_adjust(left=0.2)
 
+lines = ax.get_lines()
```

Comment:
It would be better to capture the lines are they are created rather than using `ax.get_lines()`,

```python
lines = [l0, l1, l2]
```